application/src/main/java/com/sanction/thunder/dao/DatabaseException.java

```diff
@@ -3,6 +3,11 @@
 public class DatabaseException extends RuntimeException {
   private final DatabaseError error;
 
+  /**
+   * Constructs a new DatabaseException.
+   *
+   * @param error The type of error that occurred.
+   */
   public DatabaseException(DatabaseError error) {
```

Comment:
I'm surprised Maven didn't complain about this

Reply:
@NickEckert me too!